cluster/gce/gci/configure-helper.sh

```diff
@@ -1864,6 +1864,8 @@ EOF
   fi
   if [[ "${ENABLE_METADATA_PROXY:-}" == "simple" ]]; then
     setup-addon-manifests "addons" "metadata-proxy/gce"
+    local -r metadata_proxy_yaml="${dst_dir}/metadata-proxy/gce/metadata-proxy.yaml"
+    update-prometheus-to-sd-parameters ${metadata_proxy_yaml}
```

Comment:
The same is required for the corresponding file in container-linux dir.

Reply:
Done.